job-runner/runner/runner.go

```diff
@@ -192,13 +200,19 @@ TRY_LOOP:
 		// need to increment tryNo first. At this point, we're effectively on
 		// the next try. E.g. try 1 fails, we're waiting for try 2, then we're
 		// stopped: we want try 2 state=stopped so on resume try 2 is re-run.
+		r.Lock()
 		tries++
 		tryNo++
-		totalTries++
+		r.totalTries++
+		r.retrySleep = true
+		r.Unlock()
 		select {
 		case <-time.After(r.retryWait):
+			r.Lock()
+			r.retrySleep = true
```

Comment:
`r.retrySleep = false` :wink:

Reply:
Hah thanks. This is why we do code reviews. :-)